models/task/task.go

```diff
@@ -49,20 +50,20 @@ func FindTask(listID int64, taskID int64) (*TaskStruct, error) {
 		logging.SharedInstance().MethodInfo("task", "FindTask", true).Errorf("cannot find task or list did not contain task: %v", taskID)
 		return nil, errors.New("cannot find task or list did not contain task")
 	}
-	task := NewTask(id, listId, userId, issueNumber, title, description)
+	task := NewTask(id, listId, projectId, userId, issueNumber, title, description)
 	return task, nil
 }
 
-func FindByIssueNumber(issueNumber int) (*TaskStruct, error) {
+func FindByIssueNumber(projectID int64, issueNumber int) (*TaskStruct, error) {
```

Comment:
__[golint]__
* [golint] exported function FindByIssueNumber should have comment or be unexported